cygnus-common/spec/SPECS/cygnus-common.spec

```diff
@@ -171,8 +171,15 @@ rm -rf $RPM_BUILD_ROOT
 /var/run/%{_project_name}
 
 %changelog
+* Wed Jun 02 2021 Fermin Galan <fermin.galanmarquez@telefonica.com> 2.10.0
+- [cygnus-common][Mongo] Check mongo uri format (#2046)
+- [cygnus-common][Mongo] Use mongo bulkWriter for aggregated data (#2018)
+- [cygnus-common][MySQL, PostgreSQL, Postgis] Persit error about upsert
+- [cygnus-common][MySQL] Upgrade mysql-connector from 5.1.47 to 5.1.49
+- [cygnus-common] remove hive-exec dependency (unneeded)
+- [cygnus-common] Upgrade mongodb-driver from 3.11.0 to 3.12.8
+
 * Mon May 10 2021 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.9.0
-- [cygnus-ngsi][MongoSink] Cast to mongo Date type all NGSI DateTime attributes in column and row mode (#2020)
```

Comment:
This seems to be a small issue introduced in https://github.com/telefonicaid/fiware-cygnus/pull/2024. I have taken the opportunity of this PR to fix it.

NTC (informative)